services/brig/src/Brig/User/Event.hs

```diff
@@ -1,4 +1,6 @@
 {-# LANGUAGE RecordWildCards #-}
+{-# OPTIONS_GHC -Wno-incomplete-record-updates #-}
```

Comment:
This is not test code, so maybe we can get rid of this. (we should keep the `partial-fields` one for now instead of changing the `UserEvent` type definition, though)

Reply:
I got rid of this.